### DISCUSSION TITLE
Would I get more attention on my issues if I helped as code janitor?

### DISCUSSION THREAD

**Discussion by anadon** [mesonbuild/meson#10235] [Q&A]
I'm looking to get some attention for my issue.  Looking around at other issues and outstanding issues, there seem to be many which are almost done.  Would wrapping up these almost done PRs and issues be helpful or spammy?

**Comment by eli-schwartz** [comment]
If you can verify the status of other issues, for example by determining that it is no longer an issue or that it has been fixed, that would be tremendously helpful. I semi-regularly go through the list trying to find old bugs and feature requests that I can close, but I don't always know what to look for so I probably miss lots of them.

Likewise, if you see a PR which seems like it can be completed, your feedback would be great, or if the PR author has lost interest in the feature then you are more than welcome to adopt it and push it through to completion.

Is there anything in particular you were thinking of?

**Reply by anadon** [reply]
One thing I'm seeing here is that many issues are untagged and I only really have a Linux system to test with.  Does github support fine grained enough permissions to tag issues?  That might make them easier to wade through.  If not for your purposes, then for mine.

**Comment by anadon** [comment]
Just to try and trade time for time in effect.  I'll need some thought with
my recently submitted issue so I should be fair and at least lift other
burdens in this project that are more novice friendly.  I'd imagine reviews
for new target languages are going to be harder than most PRs and I'll make
mistakes initially.
